stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -157,11 +158,15 @@ private static String getIdAttribute(View element) {
   }
 
   @Override
+  @Nullable
   public View getViewForHighlighting(View element) {
     return element;
   }
 
   @Override
+  public void getBoundsForHighlighting(View element, Rect bounds) { }
```

Comment:
The rest of the code places the closing brace on the next line, so let's maintain that throughout this PR as well